### DISCUSSION TITLE
考虑和社区共建一个 Vue 转 Taro 的方案

### DISCUSSION THREAD

**Discussion by Dante-dan** [NervJS/taro#8961] [反向转换小组]
Taro 一开始是一个类 `React` 的多端解决方案，后来才开始支持 `Vue`。 但是 Taro 一直缺少一个从旧 Vue 项目迁移到 Taro 的方案。

刚好最近在考虑这件事情，包括之前提到的 **小程序转 Taro Vue** , 都可以考虑公开文档，社区共建的方案。这也是开源的魅力所在。

**Comment by Chen-jj** [comment]
在撸文档，比想象中多。。清明过后应该能搞定

**Reply by Dante-dan** [reply]
大佬辛苦了！持续关注。

**Reply by Dante-dan** [reply]
https://github.com/NervJS/taro/discussions/9029       这个是新的想法吗？

看起来文档也不远了

**Reply by Dante-dan** [reply]
日常催更......大佬可以一版一版的放出来的  :rocket:

**Reply by IanTong** [reply]
大佬辛苦！翘首以盼，顺便问下MPvue项目直接转Taro Vue可行吗？有这个需求的人应该很多

**Comment by GPX-NSlog** [comment]
3 年了，现在有方案了吗？
